compiler/types.nim

```diff
@@ -115,7 +118,7 @@ proc isPureObject*(typ: PType): bool =
 proc isUnsigned*(t: PType): bool =
   t.skipTypes(abstractInst).kind in {tyChar, tyUInt..tyUInt64}
 
-proc getOrdValue*(n: PNode; onError = high(Int128)): Int128 =
+proc getOrdValueAux*(n: PNode): Int128Calc =
```

Comment:
I think the more common way to do this is to define this as `getOrdValue(n: PNode, error: var bool): Int128` where `error` is set to true if it errors, then define an overload that just sets `error` to a dummy variable.

We really shouldn't depend on this proc failing for logic though.

Reply:
agreed